src/elements/Link/Link.tsx

```diff
@@ -13,9 +13,7 @@ export interface LinkProps extends SpaceProps {
  * Spec: https://zpl.io/2Gm6D3d
  */
 export const Link = styled.a<LinkProps>`
-  ${space};
-  ${styledColor};
-  cursor: pointer;
```

Comment:
I think we might need cursor: pointer just to be on a safe side. We are often use `Link` with just onClick assignment. I think the underline might be lost in this case.

Reply:
This component is based on the `a` tag so it defaults to `cursor: pointer;` and javascript functions don't affect CSS, so this should be good.